utils/clean.py

```diff
@@ -97,5 +104,215 @@ def clean_state(self) -> (pd.DataFrame, pd.DataFrame, pd.DataFrame):
         Returns: use preprocess, clean, standardize, and create_tables methods
         to output (individuals_table, organizations_table, transactions_table)
         as defined in database schema
+
+        Inputs:
+            filepaths_list: list of absolute filepaths to relevant state data.
+                required naming conventions, order, and extensions
+                defined per state.
+
+        Returns: cleans the state and returns the standardized Inidividuals,
+        Organizations, and Transactions tables in a tuple
         """
+
         pass
+
+
+class ArizonaCleaner(StateCleaner):
+    """This class is based on the StateCleaner abstract class,
+    and cleans Arizona data"""
+
+    def preprocess(filepaths_list: list[str]) -> list[pd.DataFrame]:
+        """Turns filepaths into dataframes
+
+        The input must be a list of valid filepaths which lead
+        to pandas dataframes. Typically, these should be just two
+        files: a transactions file and a details file, as
+        harvested by az_curl_crawler. If these conditions are not
+        met, the rest of the pipeline will not work
+
+        args: list of two filepaths for dataframes,
+        transactions and details, in that order
+
+        returns: a list of two dataframes, transactions and details,
+        in that order
+
+        """
+
+        df_list = []
+
+        for filepath in filepaths_list:
+            df_list.append(pd.read_csv(filepath))
+
+        return df_list
+
+    def clean_state(filepaths: list[str]) -> (pd.DataFrame, pd.DataFrame, pd.DataFrame):
+        """Calls the other methods in order
+
+        This is the master function of the ArizonaCleaner
+        class, and calling it will activate the cleaning
+        pipeline which takes in filenames and outputs cleaned,
+        standardized, and schema-compliant tables
+
+        args: list of two filepaths which lead to dataframes
+
+        returns: three schema-compliant tables for
+        transactions, individuals, and organizations
+
+        """
+
+        transactions, details = ArizonaCleaner.preprocess(filepaths)
```

Comment:
we decided that the functions wouldn't call each other within another but they would be called one after another in create_tables().

for example: filepaths_lst = create_filepaths_lst()
processed_dataframes = preprocess(filepaths_lst)